### ISSUE TITLE
The train test split is both used during the training stage.

### ISSUE THREAD

**Issue by lexical-yoda** [Chang-Chia-Chi/TrackNet-Badminton-Tracking-tensorflow2#12]
The file `train.py` line 50

`history = model.fit(data_generator(BATCH_SIZE, x_train+x_test, y_train+y_test, FRAME_STACK), `

the train and test split are both used. Is that intentional, if so could you please explain why?

**Comment by Chang-Chia-Chi** [comment]
I can't recall why I added the test set when training. It's might due to a lack of training data at that time.

Anyway, I just fixed it in commit [a2784d7](https://github.com/Chang-Chia-Chi/TrackNet-Badminton-Tracking-tensorflow2/commit/a2784d7bb784b33cea6395656472be1adf3ab35c).
